src/App.js

```diff
@@ -22,8 +22,8 @@ class App extends Component {
         </NavLink>
 
         <Switch>
-          <Route path="/1stPage" component={"COMPONENT NAME HERE"}/>
-          <Route path="/2ndPage" component={"COMPONENT NAME HERE"}/>
+          <Route path="/1stPage" component={Home}/>
```

Comment:
Please set up the routes correctly here

Use '/' for home and '/chatroom' for chatroom